cdap-client/src/main/java/co/cask/cdap/client/exception/DisconnectedException.java

```diff
@@ -15,8 +15,28 @@
  */
 package co.cask.cdap.client.exception;
 
+import co.cask.cdap.client.config.ConnectionConfig;
+
+import javax.annotation.Nullable;
+
 /**
  * Thrown when the client is not connected, but a request was attempted.
  */
 public class DisconnectedException extends RuntimeException {
+
+  @Nullable
```

Comment:
is this necessary? I think just having it on the getter is sufficient, since there's no way you can access the field directly.